packages/shared/src/porter.ts

```diff
@@ -28,11 +32,24 @@ export const domains: Record<string, Domain> = {
 };
 
 export const getPorterUri = (domain: Domain): string => {
-  const uri = porterUri[domain];
+  return getPorterUris(domain)[0];
+};
+
+export const getPorterUris = (domain: Domain, porterUri?: string | string[]): string[] => {
```

Comment:
What do you think about updating this function signature like this? In-place initialization of the default value, we can also demand that the users pass in an array no matter what.
```suggestion
export const getPorterUris = (domain: Domain, porterUris: string[] = []): string[] => {
```

Reply:
It should also simplify other methods slightly

Reply:
✔️